public/css/batch.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
```

Comment:
What is difference between these 2 batch.html. and why is one under the /css folder?